web/styles/compose.css

```diff
@@ -742,63 +768,45 @@ textarea.new_message_textarea {
     }
 }
 
-textarea.new_message_textarea,
-#compose_recipient_box {
-    border: 1px solid hsl(0deg 0% 0% / 20%);
-    box-shadow: none;
-    transition: border 0.2s ease;
-    color: var(--color-text-default);
-
-    &:focus {
-        outline: 0;
-        border: 1px solid hsl(0deg 0% 67%);
-        box-shadow: none;
-    }
-}
-
 #compose_recipient_box {
     display: grid;
-    grid-template-columns: 1fr auto;
+    grid-template-columns: auto 1fr auto;
     align-items: stretch;
     flex: 1 1 0;
-    border-radius: 3px;
+    border-radius: 4px;
     background: hsl(0deg 0% 100%);
 
-    /* Give the recipient box, a `<div>`, the
-       correct styles when focus is in the
-       #stream_message_recipient_topic `<input>` */
-    &:focus-within {
-        outline: 0;
-        border: 1px solid hsl(0deg 0% 67%);
-        box-shadow: none;
-    }
-
     #stream_message_recipient_topic,
     #recipient_box_clear_topic_button {
-        background: none;
-        border: none;
+        /* background: none makes the topic input transparent in which
+        case it would not hide the edge of .narrow_to_compose_recipients
+        it overlaps, hence we use inherit instead */
+        background: inherit;
+        /* We make the borders of both these elements look like 1
+        single element's border */
```

Comment:
Generally, comments should be indented, so they don't look like bits of normal code.